orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTask.groovy

```diff
@@ -55,9 +60,9 @@ class CreateBakeTask implements RetryableTask {
   @Override
   TaskResult execute(Stage stage) {
     String region = stage.context.region
-    def bake = bakeFromContext(stage)
 
     try {
+      def bake = bakeFromContext(stage)
```

Comment:
Please drop a comment here explaining the behavior when a bake is requested for a base image that is not configured.